### DISCUSSION TITLE
Can you add Mash-up's World?

### DISCUSSION THREAD

**Discussion by FrenchWith0Skill** [Nostalgica-Reverie/Content-Monorepo#129] [Ideas]
Can you add Mash-up's World like the Christmas world or Halloween world? It will be cool if you can do this and add custom mash up world like Christmas 2024 (you make a new world with new deco).

**Comment by omo50** [comment]
mash-up worlds are planned as a resource pack in the future

**Reply by FrenchWith0Skill** [reply]
Ok thanks 👍
